.changes/v3.9.0/1007-bug-fixes.md

```diff
@@ -0,0 +1,3 @@
+* Add `prefix_length` field to `resourceVcdVappNetwork` to support creating IPv6 vApp networks [GH-1007]
```

Comment:
If this is a bug fix, we need to state what the bug is, possibly with a reference to an issue report

Reply:
Should I link the Github issue aswell, something in form of 
```
* Add `prefix_length` field to `resourceVcdVappNetwork` to support creating IPv6 vApp networks. This field is necessary as netmask representation is not suitable for IPv6  [GH-999] [GH-1007]
```

Reply:
Just noticed: `resourceVcdVappNetwork` - we should state user-facing resource name here.

Reply:
The line should be something like this.
```
* Add `prefix_length` field to `resourceVcdVappNetwork` as creating IPv6 vApp networks was not supported due to the lack of a suitable subnet representation (Issue #999) [GH-1007]
```

However, if we are making change that will affect existing scripts because of a bug, we should also state that the default value we are removing was incorrect. This is a legitimate change in semantic versioning: we can't change what we have released, but we must change what has incorrect behavior, even if it breaks user workflows.

Reply:
Updated. I'm not sure if this is clear enough or if I maybe should be less explicit in a changelog entry